packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -236,6 +237,13 @@ const SceneView = ({
   const [customHeaderHeight, setCustomHeaderHeight] =
     React.useState(defaultHeaderHeight);
 
+  const cachedAnimatedHeaderHeight = React.useRef(customHeaderHeight);
```

Comment:
we have `useAnimatedValue` hook for that

Reply:
I'll try that, thanks!

Reply:
Sorry, I linked to the wrong line.

The following:
```
const animatedHeaderHeight = React.useRef(
    new Animated.Value(customHeaderHeight, {
      useNativeDriver: true,
    })
  ).current;
````

can be replaces with `useAnimatedValue`